### PR TITLE
Create ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-57897](https://cumulocity.atlassian.net/browse/MTM-57897)
Original PR: [6458](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6458)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Smart Rule contains unexpected child MO (#6409) [GRAFT][release/cd] (#6458)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Smart rules can contain unexpected child managed objects
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57897
+version: 1020.0.10
+---
+Smart Rule contains unexpected child MO (#6409) [GRAFT][release/cd] (#6458)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md:17]: The Claude AI suggested to use the following body:
```suggestion
In some cases, smart rules could incorrectly contain child managed objects that were not expected. This caused the smart rules to behave in unintended ways and could lead to incorrect results. The issue has been resolved and smart rules now only contain the expected child managed objects. Existing smart rules that contained unexpected child managed objects will need to be reviewed and potentially updated to ensure they work as intended.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by BeateRixen** [comment]
@pmich-sag Please review, edit and approve.

File: content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md (hunk 1 above)
Comment (pmich-sag) [reply content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md:3]: I think that title does not need to be changed.

File: content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md (hunk 2 above)
Comment (pmich-sag) [reply content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md:17]: I would suggest a little bit different description.

Comment (pmich-sag) [inline content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md:17]:
```suggestion
When smart rule is created in context of parent device, during smart rule creation it is possible to select child device. When smart rule is saved it automatically becomes active for selected asset. Then on list of smart rules it is possible to see that it is active for child device.
```

Comment (pmich-sag) [reply content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md:17]: @Airdroplet what do you think? Is it ok?

File: content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md (hunk 1 above)
Comment (pmich-sag) [inline content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md:3]:
```suggestion
title: Smart Rule list contains children column when there is child assets or child devices assign to MO
```

**Review by pmich-sag — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57897
+version: 1020.0.10
+---
+When smart rule is created in context of parent device, during smart rule creation it is possible to select child device. When smart rule is saved it automatically becomes active for selected asset. Then on list of smart rules it is possible to see children column and that it is active for child device.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md:17]:
```suggestion
If a smart rule is created for a parent device, during smart rule creation you can select for which child devices the smart rule should become active. When the smart rule is saved it automatically becomes active for the selected assets. In the smart rules list,  a children column is displayed showing for which child device the smart rule is active.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Smart Rule list contains children column when there is child assets or child devices assign to MO
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md:3]:
```suggestion
title: Smart rules list now contains a children column if child assets are assigned to a managed object
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-57897
+version: 1020.0.10
+---
+If a smart rule is created for a parent device, during smart rule creation you can select for which child devices the smart rule should become active. When the smart rule is saved it automatically becomes active for the selected assets. In the smart rules list,  a children column is displayed showing for which child device the smart rule is active.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md:17]: @pmich-sag As this is a fix, what was the bug :)? Sounds more like a feature.

Comment (pmich-sag) [reply content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md:17]: Description of the [issue](https://cumulocity.atlassian.net/browse/MTM-57897) is a little bit different than what was to be fix. You can see comments there, and it turned out, that parent-device has assigned some childAsset, and children column was visible. But it was not visible for child-device which was assigned to parent-device as childDevice. Now implementation checks both of these properties.
MTM-57897_children_column

Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md:17]:
```suggestion
In the smart rules list of a group or a device, the children column, which shows for which child device the smart rule is active, did not always show up properly. This issue has been fixed and the children column is now always displayed if the smart rule is activated for a child device.
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Smart rules list now contains a children column if child assets are assigned to a managed object
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-10-smart-rule-contains-unexpected-child-mo-6409-graft-release-cd-6458.md:3]:
```suggestion
title: Smart rules list now always shows the children column properly if child assets are assigned to a managed object
```

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**